maintainers/maintainer-list.nix

```diff
@@ -16582,6 +16582,11 @@
       fingerprint = "1401 1B63 393D 16C1 AA9C  C521 8526 B757 4A53 6236";
     }];
   };
+  rosehobgoblin = {
+    name = "J. L. Bowden";
+    github = "rosehobgoblin";
+    githubId = "84164410";
```

Comment:
this needs to be an integer 
https://gist.github.com/GrahamcOfBorg/14356eb528c57a9d7fa7588d2b51e087